charts/generic-prometheus-alerts/templates/aws-sqs-alerts.yaml

```diff
@@ -18,14 +19,21 @@ spec:
   groups:
     - name: {{ $targetApplication }}-sqs
       rules:
+        - record: business_day
+          expr: 0 < day_of_week() < 6
+        - record: business_hour
+          expr: 7= < hour() < 18
```

Comment:
between 7am and 6pm (8am to 7pm BST)

Reply:
"7=" looks odd, does this mean inclusive?

Reply:
🤦